pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Topic.java

```diff
@@ -110,7 +110,7 @@ default long getOriginalSequenceId() {
 
     CompletableFuture<Void> checkReplication();
 
-    CompletableFuture<Void> close();
+    CompletableFuture<Void> close(boolean force);
```

Comment:
Apart from the bool vs enum discussion, why do we need 2 different behaviors? Can't we just consider "force" the only approach?

Reply:
> Can't we just consider "force" the only approach?

No, because first we want to close topic gracefully by closing all clients first and then close managed-ledger. If things don't get closed gracefully then close topic forcefully by closing managed-ledger before giving up ownership of the bundle. So, we need both behavior.